scripts/newsexsystem.gd

```diff
@@ -2242,9 +2287,14 @@ func endencounter():
 			###---Added by Expansion---### Removed , added \n
 			text += "\n[color=aqua]Desires fullfiled: " + str(i.requestsdone) + '[/color]'
 			###---End Expansion---###
+		if i.person.race_display == "Succubus": #ralphC - Succubus orgasms don't produce mana
+			#print("Ralph Test: Trying to set mana from Succubus's orgasm to zero. Mana before attempt:" + str(mana) + " mana") #ralphC temp
+			mana -= mana  #ralphC
```

Comment:
Please don't use such obscure code, simply set mana to zero.

Reply:
comment incorporated for next commit